src/ciel/main.go

```diff
@@ -1,16 +1,15 @@
 package main
 
 import (
-	"ciel-driver"
 	"fmt"
 	"log"
 	"math/rand"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"strings"
-	"syscall"
 	"time"
+
+	"ciel-driver"
```

Comment:
Actually by convention and as used by the official `goimports` tool, the third-party imports should be separated by golang core libraries.

Goimports: [golang.org/x/cmd/goimports](https://github.com/golang/tools/tree/master/cmd/goimports)